steembase/http_client.py

```diff
@@ -169,12 +184,17 @@ def call(self,
             transaction.  In latter case, the exception is **re-raised**.
 
         """
-        body = HttpClient.json_rpc_body(name, *args, api=api, kwargs=kwargs)
+
+        api = kwargs.get('api', None)
+        return_with_args = kwargs.get('return_with_args', None)
+        _ret_cnt = kwargs.get('_ret_cnt', 0)
+
+        body = HttpClient.json_rpc_body(name, *args, kwargs=kwargs)
         response = None
+
         try:
             response = self.request(body=body)
-        except (MaxRetryError, ConnectionResetError, ReadTimeoutError,
-                RemoteDisconnected, ProtocolError) as e:
+        except (MaxRetryError, ReadTimeoutError, ProtocolError) as e:
```

Comment:
This is an important method and exceptions are a key part of its flow, so I'm concerned removing caught exceptions here will change its behavior.

Reply:
`RemoteDisconnected` and `ConnectionResetError` are both Python 3 only exceptions. Not sure how to capture them in 2.7.